maat-court-data-api/src/main/java/gov/uk/courtdata/dces/service/FdcContributionsService.java

```diff
@@ -52,26 +54,24 @@ public class FdcContributionsService {
     private final DebtCollectionService debtCollectionService;
     private final FdcContributionMapper fdcContributionMapper;
 
-    private static final Function<FdcContributionsEntity, FdcContributionEntry> BUILD_FDC_ENTRY =
```

Comment:
This method is missing some fields like judApportionPercent, status etc which were always being returned as null. There is already a suitable mapper for this which does a better job and should be used instead.